src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -21,9 +21,9 @@ public String index() {
         return "index";
     }
 
-    @RequestMapping("/AddFacility.html")
+    @RequestMapping("/facility/add")
```

Comment:
I changed this endpoint. Here 'add' is subset of 'facility' to scale the app better

Reply:
Ok, yeah, that makes sense.
Also, endpoints tend to be all lowercase.